lib/active_merchant/billing/gateway.rb

```diff
@@ -316,6 +316,14 @@ def split_names(full_name)
         [first_name, last_name]
       end
 
+      def clean_emoji_emoticons_and_accent(str = '')
+        str = str.force_encoding('utf-8').encode
+        emoji_regex = /[\u{1f600}-\u{1f64f}\u{2702}-\u{27b0}\u{1f680}-\u{1f6ff}\u{24C2}-\u{1F251}\u{1f300}-\u{1f5ff}]/
+        emoticon_regex = /[:;=8xX]-?[)(DPpO3]/
+        str = str.gsub(emoji_regex, '').gsub(emoticon_regex, '').strip
+        I18n.transliterate(str)
```

Comment:
Looks like you changed it since yesterday. This definitely works. For MercadoPago just works `I18n.transliterate(name)` also works. It just ends up replacing the emoji with some question marks.

Reply:
I think I prefer not to have the question mark appear because sometimes the gateway verifies the name printed on the card.